tornado-drivers/spirv/src/main/java/uk/ac/manchester/tornado/drivers/spirv/SPIRVBackend.java

```diff
@@ -230,8 +230,8 @@ public SPIRVDeviceContext getDeviceContext() {
         return this.context;
     }
 
-    public void reset() {
-        getDeviceContext().reset();
+    public void reset(long executionPlanId) {
```

Comment:
Why the `PTXBackend` does not have such a method?

Reply:
This method was never triggered.